LoopKitUI/PumpManagerUI.swift

```diff
@@ -44,7 +44,12 @@ public protocol PumpStatusIndicator {
 
 public typealias PumpManagerViewController = (UIViewController & PumpManagerOnboarding & CompletionNotifying)
 
-public protocol PumpManagerUI: DeviceManagerUI, PumpStatusIndicator, PumpManager, DeliveryLimitSettingsTableViewControllerSyncSource, BasalScheduleTableViewControllerSyncSource {
+public typealias SyncSchedule = (_ items: [RepeatingScheduleValue<Double>], _ completion: @escaping (Result<BasalRateSchedule, Error>) -> Void) -> Void
```

Comment:
If these typealiases are just used by the settings UI and its callers, they should be defined there.

Reply:
I've put them in the `TherapySettingsViewModel`. If that is not a good spot, where are you thinking?